labs/query_enriched_event_data_with_spark/README.md

```diff
@@ -0,0 +1,65 @@
+# Querying Optimizely Enriched Event data with Apache Spark
+
+In the [Enriched Events](../enriched_events) Lab we learned how to download [Enriched Event Data](https://docs.developers.optimizely.com/web/docs/enriched-events-export).  In this Lab we'll learn how to query that data with [Apache Spark](https://spark.apache.org/).
+
+Spark is a powerful, widely-adopted engine for data processing. It's easy to run on a single machine and scales up to handle arbitrarily large workloads. It also works well with the Enriched Events dataset, which is stored using the [Apache Parquet](https://parquet.apache.org/) format.
+
+In this Lab, we'll assume you're working with a standalone Spark cluster running on your computer.  However, this and other Lab notebooks can be modified to work with remote Spark clusters as well.
+
+
+## Running PySpark locally with Docker
+
+The simplest way to get started with PySpark is to run it in a [Docker](https://www.docker.com/) container.  You use Docker to run the Lab notebook with a single command:
+
+```sh
+docker run -it --rm \
+    -p 8888:8888 \
+    -v $(pwd):/home/jovyan \
+    jupyter/pyspark-notebook \
+    jupyter lab query_enriched_event_data_with_spark.ipynb
```

Comment:
Should we indicate how to get to this file? Or do you think there should be some kind of generic "Working with labs" instruction that indicates to users that they should download the resources file and unzip it into a convenient location?

Reply:
oh, good question.  Are we still planning on linking to the zipped lab directory somewhere?  If so, I could make a reference to that.

Reply:
Yeah, currently, there's a download link to a 'Resources' button that will download a zip of the contents.

But now thinking through this, any reason you prefer the zip file rather than pointing to a branch of this github repository? That would reduce the need for people to know how to download the zip file, but does require people to know how to clone a github repository. Do you think more people know how to handle the former?

Reply:
Zip file is less overhead and easier for a customer to edit in order to solve their own problem.  If I clone the git repo and modify a notebook to work with my own data (or even just re-run a notebook on the included dataset) then every thing I do shows up as unstaged changes in the repo, which is annoying.

I do think it's useful to refer back to the repo--could we link to the zip file as well as the directory in the GH repo under "Resources"?